graylog2-server/src/main/java/org/graylog2/shared/rest/resources/system/RemoteDataNodeStatusResource.java

```diff
@@ -33,4 +36,9 @@ public interface RemoteDataNodeStatusResource {
     @GET("/logs/stderr")
     Call<List<String>> opensearchStdErr();
 
+    @Streaming
```

Comment:
: We could remove the proxied calls to `/logs/stdout` and `/logs/stderr` as they are not used any more

Reply:
removed, thx!